deploy-nagios/templates/nagios-template.j2

```diff
@@ -0,0 +1,256 @@
+                    "value": "${NAGIOS_PASSWORD}"
+                  },
+                  {
+                    "name": "RHMAP_ROUTER_DNS",
+                    "value": "${RHMAP_ROUTER_DNS}"
+                  },
+                  {
+                    "name": "RHMAP_HOSTGROUPS",
+                    "value": "${RHMAP_HOSTGROUPS}"
+                  },
+                  {
+                    "name": "MAC_SERVERS",
+                    "value": "${MAC_SERVERS}"
+                  }
+                ],
+                "image": "${NAGIOS_IMAGE}",
+                "name": "nagios",
+                "ports": [
+                  {
+                    "containerPort": 8080
+                  }
+                ],
+                "volumeMounts": [
+                  {
+                    "name": "nagios-data-volume",
+                    "mountPath": "/var/log/nagios"
+                  }
+                ],
+                "resources": {
+                  "limits": {
+                    "cpu": "800m",
+                    "memory": "800Mi"
+                  },
+                  "requests": {
+                    "cpu": "200m",
+                    "memory": "200Mi"
+                  }
+                },
+                "readinessProbe": {
+                  "failureThreshold": 3,
+                  "initialDelaySeconds": 5,
+                  "periodSeconds": 10,
+                  "successThreshold": 1,
+                  "tcpSocket": {
+                    "port": 8080
+                  },
+                  "timeoutSeconds": 5
+                },
+                "livenessProbe": {
+                  "failureThreshold": 3,
+                  "initialDelaySeconds": 60,
+                  "periodSeconds": 60,
+                  "successThreshold": 1,
+                  "tcpSocket": {
+                    "port": 8080
+                  },
+                  "timeoutSeconds": 5
+                }
+              }
+            ],
+            "serviceAccount": "nagios"
+          }
+        },
+        "triggers": [
+          {
+            "type": "ConfigChange"
+          }
+        ]
+      }
+    }
+  ],
+  "parameters": [
+    {
+      "description": "SMTP server address to use for sending alert emails e.g. 1.2.3.4:25",
+      "name": "SMTP_SERVER",
+      "value": "localhost"
+    },
+    {
+      "name": "SMTP_USERNAME",
+      "description": "SMTP username",
+      "value": "username"
+    },
+    {
+      "name": "SMTP_PASSWORD",
+      "description": "SMTP password",
+      "value": "password"
+    },
+    {
+      "description": "Use TLS for message delivery (auto|yes|no)",
+      "name": "SMTP_TLS",
+      "value": "auto"
+    },
+    {
+      "description": "From email address for alerts",
+      "name": "SMTP_FROM_ADDRESS",
+      "value": "admin@example.com"
+    },
+    {
+      "description": "To email address for alerts",
+      "name": "RHMAP_ADMIN_EMAIL",
+      "value": "root@localhost"
+    },
```

Comment:
let's make the params for sending emails configurable. This way the admin user can specify the values during the installation, rather than having to update them again via OpenShift after the installation.